doc/ChangeLog

```diff
@@ -94,24 +87,24 @@ Changes answers relative to baseline:
     - what code configurations: ALL
     - what platforms/compilers: ALL
     - nature of change: roundoff
-    This change occurs because the parameter values for
+    Changes are due to four parameters:
  accum_factor 
  ko25_coef 
  kc25_coef 
  frac_sat_soil_dsl_init 
-    were not explicitly specified as _r8 when hardwired in the code,
-    while they are now declared "double" in the params.nc files. I
+    that were not declared "_r8" when hardwired in the code,
+    while now they are declared "double" in the params.nc files. I
     showed that the change was no greater than roundoff by writing
     the new and old values to the lnd and cesm log files and confirming
     differences smaller than 1.e-6.
 
```

Comment:
@ekluzek and @billsacks 
I wanted to confirm that you're ok with my statement here:
Differences start less than 1e-6 when the 4 parameters get read in.
Do these qualify as roundoff differences?

Reply:
Yes, though I'd probably clarify that they are single-precision roundoff-level differences. (Typical roundoff-level differences are double-precision roundoff-level, at the 1e-15 level rather than the 1e-6 level.)